re.rizin.cutter.yml

```diff
@@ -0,0 +1,80 @@
+app-id: re.rizin.cutter
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.15-21.08'
+branch: stable
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: /bin/bash
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --device=dri
+  - --allow=devel
+
+modules:
+  #stolen from https://github.com/flathub/org.freecadweb.FreeCAD/blob/master/org.freecadweb.FreeCAD.yaml
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      prepend-path: /usr/lib/sdk/llvm12/bin
+      prepend-ld-library-path: /usr/lib/sdk/llvm12/lib
```

Comment:
Since this repeats in almost every module, you probably should set these `build-options` globally.

Reply:
Yeah looks like the smart thing to do ^^